modules/administration/pages/content-lifecycle.adoc

```diff
@@ -38,8 +38,8 @@ The project defines the software channel sources, the filters used to find packa
 . In the [guimenu]``Filters`` dialog, select the filters you want to attach to the project.
     To create a new filter, click btn:[Create new Filter].
 . Click btn:[Add Environment].
-. In the [guimenu]``Environment Lifecycle`` dialog, give the first environment a name and a description, and click btn:[Save].
-    The [guimenu]``Name`` field only accepts lowercase letters, numbers, periods, hyphens, and underscores.
+. In the [guimenu]``Environment Lifecycle`` dialog, give the first environment a name, a label and a description, and click btn:[Save].
```

Comment:
```suggestion
. In the [guimenu]``Environment Lifecycle`` dialog, give the first environment a name, a label, and a description, and click btn:[Save].
```

Reply:
Yes, I'm *that* person :wink:

Reply:
rofl "oxford comma alert!

Reply:
Most style guides recommend using the Oxford comma, because US English tends to prefer it. So it's generally good practice :wink:

Reply:
Good to know, thanks!